worker/src/Worker.cpp

```diff
@@ -84,7 +87,9 @@ void Worker::Close()
 	this->channel->Close();
 
 	// Close the PayloadChannel.
-	this->payloadChannel->Close();
+	// Commenting out this cleanup of payloadChannel because it is not setup
```

Comment:
we should specify that it was commented out as part of the effort to add support for unix socket
in order to support the unix sockets (replacing pipe) we use the custom callback functions (add description of the constructor being used) and since we are not providing custom functions to deal with payload data we need to comment this out